packages/login/src/views/resetCredentialsForm/forgottenItemForm.tsx

```diff
@@ -58,7 +63,9 @@ class ForgottenItemComponent extends React.Component<Props, State> {
   }
 
   render() {
-    const { intl, goToHome } = this.props
+    const { intl, goToHome, applicationName } = this.props
+    if (applicationName) document.title = applicationName
```

Comment:
@CornelMaze Although this should work, if you think about it the document title would be set each time the component re-renders which we do not want. We should go with a similar approach used in #4233.

Reply:
@Zangetsu101 Yeah. That's true.